R/plotly.R

```diff
@@ -119,6 +120,10 @@ plot_ly <- function(data = data.frame(), ..., type = NULL,
   id <- new_id()
   # avoid weird naming clashes
   plotlyVisDat <- data
+  # automatically namespace source
+  if (!is.null(session)) {
+    source <- session$ns(source)
```

Comment:
I was thinking namespacing would be imposed by default by plotly, but I think I'm now in favor of the approach you describe below